eternalcore-core/src/main/java/com/eternalcode/core/feature/jail/JailCommand.java

```diff
@@ -0,0 +1,255 @@
+package com.eternalcode.core.feature.jail;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.annotations.scan.feature.FeatureDocs;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.notice.NoticeService;
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+
+import java.time.Duration;
+
+@Command(name = "jail")
+@FeatureDocs(
+    name = "Jail",
+    permission = { "eternalcore.jail.setup", "eternalcore.jail.detain", "eternalcore.jail.release", "eternalcore.jail.bypass" },
+    description = "Permissions allow for you to setup jail location on your server, detain players, release players, and bypass jail."
+)
+public class JailCommand {
+
+    private final JailService jailService;
+    private final NoticeService noticeService;
+    private final PrisonerService prisonerService;
+
+    @Inject
+    JailCommand(JailService jailService, NoticeService noticeService, PrisonerService prisonerService) {
+        this.jailService = jailService;
+        this.noticeService = noticeService;
+        this.prisonerService = prisonerService;
+    }
+
+    @Execute(name = "setup")
+    @Permission("eternalcore.jail.setup")
+    @DescriptionDocs(description = "Define jail spawn area")
+    void executeJailSetup(@Context Player player) {
+        this.noticeService.create()
+            .notice(translation -> (this.jailService.isLocationSet() ? translation.jailSection().jailLocationOverride() : translation.jailSection().jailLocationSet()))
+            .player(player.getUniqueId())
+            .send();
```

Comment:
This code is so... long:
```suggestion
        this.noticeService.create()
            .notice(translation -> (this.jailService.isLocationSet()
                ? translation.jailSection().jailLocationOverride()
                : translation.jailSection().jailLocationSet()))
            .player(player.getUniqueId())
            .send();
```